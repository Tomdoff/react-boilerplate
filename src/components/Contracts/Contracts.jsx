import React, { Component } from 'react';

class Contracts extends Component {
    render () {
        return <div>Contracts</div>;
    }
}

export default Contracts;
