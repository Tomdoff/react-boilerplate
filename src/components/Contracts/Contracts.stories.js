import React from 'react';
import { storiesOf } from '@storybook/react';
import Contracts from './Contracts';
storiesOf('Contracts', module).add('Contracts', () => (
    <div>
        <Contracts />
    </div>
));
