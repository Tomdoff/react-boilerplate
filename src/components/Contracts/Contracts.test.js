import React from 'react';
import { shallow } from 'enzyme';
import Contracts from './Contracts';

describe('<Contracts />', () => {
    test('renders', () => {
        const wrapper = shallow(<Contracts />);
        expect(wrapper).toMatchSnapshot();
    });
});
