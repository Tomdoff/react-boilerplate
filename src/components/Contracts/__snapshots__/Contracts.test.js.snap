// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contracts /> renders 1`] = `ShallowWrapper {}`;
